tests/gcp_iam_roles/dynatrace-gcp-function-e2e-deployment-role.yaml

```diff
@@ -0,0 +1,43 @@
+title: E2E tests dynatrace-gcp-function
+description: Role for Dynatrace GCP function E2E deployemnt
+stage: GA
+includedPermissions:
+  -  container.clusters.get
+  -  container.configMaps.create
+  -  container.configMaps.delete
+  -  container.configMaps.get
+  -  container.deployments.create
+  -  container.deployments.delete
+  -  container.deployments.get
+  -  container.namespaces.create
+  -  container.namespaces.get
+  -  container.pods.list
+  -  container.secrets.create
+  -  container.secrets.delete
+  -  container.secrets.get
+  -  container.secrets.list
+  -  container.secrets.update
+  -  container.serviceAccounts.create
+  -  container.serviceAccounts.delete
+  -  container.serviceAccounts.get
+  -  logging.sinks.create
```

Comment:
sink permissions are not needed for helm deployment itself, should be moved to e2e-test role